Classes/Command/UpdateQbankFileStatusCommand.php

```diff
@@ -0,0 +1,124 @@
+ * LICENSE.txt file that was distributed with this source code.
+ *
+ * The TYPO3 project - inspiring people to share!
+ */
+
+namespace Pixelant\Qbank\Command;
+
+use Pixelant\Qbank\Repository\MediaRepository;
+use Pixelant\Qbank\Repository\QbankFileRepository;
+use Pixelant\Qbank\Service\QbankService;
+use QBNK\QBank\API\Model\MediaResponse;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use TYPO3\CMS\Core\Core\Bootstrap;
+use TYPO3\CMS\Core\Utility\GeneralUtility;
+
+/**
+ * Update status on QBank files.
+ * Fetches qbank files where tx_qbank_status_updated_timestamp isn't updated within interval.
+ * Should then update tx_qbank_remote_change_timestamp and tx_qbank_status_updated_timestamp.
+ * Next Command can then fetch qbank files where tx_qbank_remote_change_timestamp is
+ * newer than the tx_qbank_file_timestamp or tx_qbank_metadata_timestamp.
+ */
+class UpdateQbankFileStatusCommand extends Command
+{
+    /**
+     * Configure the command by defining the name, options and arguments.
+     *
+     * @return void
+     */
+    public function configure(): void
+    {
+        $this->setDescription('Update status of downloaded QBank files.')
+            ->addOption(
+                'limit',
+                'l',
+                InputOption::VALUE_OPTIONAL,
+                'Limit number of files to check per run. (default 10)',
+                10
+            )
+            ->addOption(
+                'interval',
+                'i',
+                InputOption::VALUE_OPTIONAL,
+                'Interval (in seconds) until files are checked again (default 86400).',
+                86400
+            )
+            ->addOption(
+                'check',
+                'c',
+                InputOption::VALUE_NONE,
+                'Only display information, don\'t update records.'
+            );
+    }
+
+    /**
+     * Executes the command for adding or removing the lock file.
+     *
+     * @param InputInterface $input
+     * @param OutputInterface $output
+     * @return int
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        Bootstrap::initializeBackendAuthentication();
+        $io = new SymfonyStyle($input, $output);
+
+        $isTestOnly = $input->getOption('check');
+        $limit = (int)$input->getOption('limit');
+        $interval = (int)$input->getOption('interval');
+
+        /** @var QbankFileRepository $qbankFileRepository */
+        $qbankFileRepository = GeneralUtility::makeInstance(QbankFileRepository::class);
+        $updateQueueu = $qbankFileRepository->fetchStatusUpdateQueue($limit, $interval);
+
+        $io->title('Update QBank file status.');
+
+        if (count($updateQueueu) === 0) {
```

Comment:
Typo in variable name. Replace in entire file.

```suggestion
        if (count($updateQueue) === 0) {
```